test/instance_files/base_instance.lua

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tarantool
+local workdir = os.getenv('TARANTOOL_WORKDIR')
+local listen = os.getenv('TARANTOOL_LISTEN')
+
+box.cfg({
+    work_dir = workdir,
+--     listen = 'localhost:3310'
+    listen = listen
+})
+
+box.schema.user.grant('guest', 'read,write,execute,create', 'universe')
```

Comment:
Some of our tests use privileges extensively, e.g. grant and revoke some privileges to `guest`.

Correct me if I'm wrong. Looks like you have to grant all these privileges to 'guest', so that cluster's netbox connection to it can call `eval` and basically control the instance. Something might break if I revoke `execute` from `guest` in the test.

Test-run connects via a special admin console:
```lua
require('console').listen(os.getenv('ADMIN'))
```
Is it possible to do the same for luatest ? If it is, all these grants wouldn't be needed.